### PR TITLE
added README.md

### PR DESCRIPTION
**Pull request by ahangarha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# About
+
+This is a demo implementation of a federated blogging system on top of
+ActivityInn framework which itself is based on ActivityPub.
+
```
Comment (MCSH) [inline README.md]: Could you link to activityinn as well?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Notice that this project is far from production. In fact, this repo is a sample
```
Comment (MCSH) [inline README.md]: I would not say it's far from production, however the rest is completely right :D

File: README.md (hunk 1 above)
Comment (ahangarha) [reply README.md]: sure

File: README.md (hunk 2 above)
Comment (ahangarha) [reply README.md]: ok

**Comment by ahangarha** [comment]
I close this PR and make another

**Pull request closed without merging**